### PR TITLE
Update Inbox Placement API repsonse

### PR DESCRIPTION
**Pull request by marco258**

### CODE REVIEW

File: source/samples/fields-ibp-seed-lists.rst [hunk 1]
```diff
@@ -12,4 +12,5 @@ name                     string       The name of the seed list
 seed_filter              string       A regular expression value that will be used to filter the list of seeds in the seed list.
 mailing_list             string       A mailing list that contains the target email, and available seeds.
 delivery_stats           object       An object that contains sub-objects that describe delivery stats. See below.
+results                  array        An array of tests performed on the seed list.
```
Comment (roaet) [inline source/samples/fields-ibp-seed-lists.rst:15]: `An array of tests performed` is a little weird. It's more like test results using from this seed list. or results from the seed list's tests.

File: source/samples/get-results.rst [hunk 2]
```diff
@@ -2,7 +2,7 @@
 .. code-block:: bash
 
   curl -s --user 'api:YOUR_API_KEY' -G \
-      https://api.mailgun.net/v4/results
+      https://api.mailgun.net/v4/inbox/results
```
Comment (roaet) [inline source/samples/get-results.rst:5]: whew good catch. 👍🏼

**Review by roaet — CHANGES_REQUESTED** [review]
Just the little changes would be good.

File: source/samples/fields-ibp-seed-lists.rst [hunk 3]
```diff
@@ -12,4 +12,5 @@ name                     string       The name of the seed list
 seed_filter              string       A regular expression value that will be used to filter the list of seeds in the seed list.
 mailing_list             string       A mailing list that contains the target email, and available seeds.
 delivery_stats           object       An object that contains sub-objects that describe delivery stats. See below.
+results                  array        An array of results from the seed list's test.
```
Comment (roaet) [inline source/samples/fields-ibp-seed-lists.rst:15]: nit: `test` -> `tests`. not sure if that is correct grammar though

**Review by roaet — APPROVED** [review]
⛄

File: source/samples/fields-ibp-seed-lists.rst (hunk 3 above)
Comment (marco258) [reply source/samples/fields-ibp-seed-lists.rst]: I believe `tests` is correct, updated. :)

**Pull request merged**